meson.build

```diff
@@ -373,12 +377,21 @@ else
     libass_for_tools = libass
 endif
 
-if get_option('test')
+if get_option('test').enabled()
     subdir('test')
 endif
-if get_option('profile')
+if get_option('compare').require(default_library != 'shared').enabled()
+    subdir('compare')
+endif
+if get_option('profile').enabled()
     subdir('profile')
 endif
+if get_option('fuzz').enabled()
+    subdir('fuzz')
+endif
+if get_option('checkasm').require(enable_asm and default_library != 'shared').allowed()
+    subdir('checkasm')
+endif
```

Comment:
So to enable libass tests, user has to enable 4 different flags? Why? Could there be single a single flag? It's unclear to me, the user, how to test libass before deployment.

Reply:
This is targeting parity with the autoconf build; simplifying the flags would be reasonable, but is out-of-scope for this PR.

Reply:
Ok, clear. So how should I test libass? Is `-Dtest=enabled` good enough as integration tests or should I enable all other flags too?

Reply:
This repository doesn’t contain any integration tests; only the infrastructure.

These flags control which utilities are built. `checkasm` can be run to verify that our assembler code matches our C code and is run by `meson test`/`make check`. The others are special-purpose utilities that can be run by hand; in particular, `test` just takes an ASS file, renders it onto a blank canvas and saves it as a PNG.

Actual integration tests are in the dedicated libass-tests repository, which contains a shell script to run them using these utilities: `compare` for regression tests and `fuzz` for crash tests.

So if you want to run all tests, you must check out the libass-tests repo, build libass with `checkasm`, `compare` and `fuzz` (but not necessarily `test`) enabled, and either
* set `-Dart-samples=path/to/libass-tests` in Meson and run `meson test` (which, at the current state of this PR, will run the tests but only give a summary verdict for all of them together)
* or run `meson test` without defining `art-samples` (which will only run `checkasm`) and run libass-tests separately (which will let you see individual test results).

Reply:
Thank you for explanation, I will see if this is feasible for me to integrate on each build.